### ISSUE TITLE
Zoom in/Zoom out not working when setScaleType = ScaleType.FIT_CENTER

### ISSUE THREAD

**Issue by wise156** [chrisbanes/PhotoView#308]
at default it was set as ScaleType.MATRIX and image enable to zoom in/out perfectly.

but when i setScaleType = ScaleType.FIT_CENTER image will unable to zoom in/out.

so can anyone help me to solve this issue pls.

thanks :)

**Comment by ibrahim132** [comment]
Library supports only "ScaleType.MATRIX" scale type.

**Comment by bejibx** [comment]
@ibrahim132 this is not true. PhotoView supports all scale types from imageView, it uses scale types when calculate base transformation matrix, see [updateBaseMatrix(Drawable)](https://github.com/chrisbanes/PhotoView/blob/master/library/src/main/java/uk/co/senab/photoview/PhotoViewAttacher.java#L866) method. It sets scale type ScaleType.MATRIX only after base matrix is calculated.

@wise156 can you post your code so we can figure out your problem?

**Comment by mavsforlife** [comment]
how can i change ScaleType?

**Comment by bejibx** [comment]
You should do this by setting scaleType attribute.
